tests/classes/potentialSet.cpp

```diff
@@ -63,4 +64,154 @@ TEST(PotentialSetTest, ComplexAddition)
     EXPECT_EQ(2.0, pots.potentialMap()["A-D"].potential.value(0));
 }
 
+TEST(PotentialSetTest, Averaging)
+{
+    GenericList moduleData;
+    auto originalPotentialsObject =
+        moduleData.realiseIf<PotentialSet>("PotentialSet", "module", GenericItem::InRestartFileFlag);
+
+    std::string filename_{"test_restart.txt"};
```

Comment:
Being _really_ picky, but there shouldn't be an underscore on this local variable.
```suggestion
    std::string filename{"test_restart.txt"};
```